src/database.ts

```diff
@@ -460,6 +602,8 @@ class Database extends ServiceObject {
       }
     });
   }
+  delete(): Promise<BasicResponse>;
```

Comment:
I believe there is a type called `IEmpty` that we should be using here instead

Reply:
same method in base class it give below Error:
`Property 'delete' in type 'Database' is not assignable to the same property in base type 'GrpcServiceObject'.
  Type '{ (): Promise<Empty>; (callback: DropDatabaseCallback): void; }' is not assignable to type '{ (): Promise<[Response]>; (callback: RequestCallback): void; }'.`